src/audio/mixer/mixer.c

```diff
@@ -470,7 +470,10 @@ static const struct comp_driver comp_mixer = {
 	},
 };
 
-static SHARED_DATA struct comp_driver_info comp_mixer_info = {
+#ifndef UNIT_TEST
```

Comment:
Why only mixer component needs these additions? Is there something in other components or their unit tests that helps to avoid to do these.